www/css/artwork.css

```diff
@@ -0,0 +1,418 @@
+}
+
+.artworks nav > a[href]:active{
+	top: 2px;
+	left: 2px;
+	box-shadow: none;
+}
+
+.artworks nav > a:last-child::after{
+	font-family: "Fork Awesome";
+	content: "\f061";
+	transition: all 200ms ease;
+	position: relative;
+	left: 0;
+	margin-left: .5rem;
+}
+
+.artworks nav > a:last-child[href]:hover::after{
+	left: .25rem;
+	position: relative;
+	transition: all 200ms ease;
+}
+
+.artworks nav > a:first-child:before{
+	font-family: "Fork Awesome";
+	content: "\f060";
+	transition: all 200ms ease;
+	position: relative;
+	right: 0;
+	margin-right: .5rem;
+}
+
+.artworks nav > a:first-child[href]:hover::before{
+	right: .25rem;
+	position: relative;
+	transition: all 200ms ease;
+}
+
+.artworks nav > a:last-child[href]:hover::after{
+	left: .25rem;
+	position: relative;
+	transition: all 200ms ease;
+}
+
+main.artworks nav ol li a:hover,
+main.artworks nav ol li.highlighted a:hover,
+.artworks nav > a:hover{
+	background-color: var(--button-highlight);
+}
+
+.artworks nav > a:not([href]){
+	cursor: default;
+	color: #efefef;
+	box-shadow: none;
+	background: #bbb url("/images/stripes-dark.svg");
+}
+
+ol.artwork-list.list{
+	display: flex;
+	flex-direction: column;
+	gap: 0;
+	margin-bottom: 4rem;
+	margin-left: auto;
+	margin-right: auto;
+	width: 100%;
+	max-width: 40rem;
+}
+
+ol.artwork-list.list > li{
+	border-radius: .25rem;
+	display: grid;
+	grid-template-columns: 16rem 1fr;
+	grid-column-gap: 2rem;
+	grid-template-rows: auto auto 1fr;
+	padding: 1rem;
+}
+
+ol.artwork-list.list > li.in_use{
+	background-color: rgba(0, 0, 0, .2);
+	background-image: url("/images/stripes.svg");
+}
+
+ol.artwork-list.list > li + li {
+	margin-top: 1rem;
+}
+
+ol.artwork-list.list > li .thumbnail-container{
+	grid-row: 1 / span 3;
+	margin-left: auto;
+}
+
+ol.artwork-list.list > li p{
+	text-align: left;
+}
+
+ol.artwork-list > li p{
+	margin: 0;
+}
+
+ol.artwork-list > li img{
+	box-sizing: border-box;
+	max-width: 100%;
+	height: auto;
+	border: 1px solid var(--border);
+	border-radius: .25rem;
+}
+
+ol.artwork-list > li > p:nth-of-type(1) > span{
+	font-weight: bold;
+}
+
+ol.artwork-list > li .author{
+	font-style: italic;
+}
+
+main.artworks nav ol{
+	list-style: none;
+	display: flex;
+	margin: 0 .5rem;
+}
+
+main.artworks nav{
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	max-width: none;
+}
+
+main.artworks nav ol li{
+	margin: 0;
+	white-space: nowrap;
+}
+
+main.artworks nav ol li a{
+	font-size: 1rem;
+	line-height: 1.4;
+	padding: 1rem;
+	margin: 0 .5rem;
+	border-radius: .25rem;
+	font-variant-numeric: normal;
+	height: calc(1.4rem + 2rem + 2px);
+	display: inline-block;
+	box-sizing: border-box;
+}
+
+main.artworks nav ol li.highlighted a{
+	background: var(--button);
+	border: 1px solid rgba(0, 0, 0, .5);
+}
+
+main.artworks nav ol li.highlighted a,
+main.artworks nav ol li a:hover{
+	color: #fff;
+	text-decoration: none;
+	text-shadow: 1px 1px 0 rgba(0, 0, 0, .5);
+}
+
+input[type="number"],
```

Comment:
I'm not sure that it's very useful to move generic stuff like this out of `core.css`/`dark.css`. Now we have identical styles in two places, and they'll both need to be updated to stay consistent -- if just the `input[type="text"]` in `core.css` gets updated then we'll have differently-styled inputs in the artwork form. 

Also, if someone wants to use number or url inputs elsewhere in the site, they'd have to either move these changes back or recreate them in the main files.

Reply:
Updated, thanks for weighing in. I deliberated on this, too, and decided you're right. It also leads fewer lines if CSS overall, which is good.